blockchain/tokensMetadata.ts

```
@@ -13,24 +27,67 @@ export function getToken(tokenSymbol: TokenSymbolType): TokenMetadataType {
   return Object.keys(tokensBySymbol).includes(tokenSymbol) ? getToken(tokenSymbol) : undefined
 }
 
+/**
+ * Retrieves the metadata for an array of token symbols.
+ *
+ * @param {TokenSymbolType[]} tokenSymbol - An array of token symbols to retrieve metadata for.
+ * @returns {typeof tokens} An array of token metadata corresponding to the provided token symbols.
+ * @throws {Error} If the provided tokenSymbol is not an array.
+ */
 export function getTokens(tokenSymbol: TokenSymbolType[]): typeof tokens {
   if (tokenSymbol instanceof Array) {
```

Biome

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)